components/org.wso2.carbon.identity.application.authenticator.samlsso/src/test/java/org/wso2/carbon/identity/application/authenticator/samlsso/TestUtils.java

```diff
@@ -54,4 +104,155 @@ public static KeyStore loadKeyStoreFromFileSystem(String keyStorePath, String pa
             throw new SecurityException(errorMsg, e);
         }
     }
+
+    public static String buildRequest(boolean isLogout, RequestData requestData) throws IOException, MarshallingException {
```

Comment:
why can't we simply use buildAuthnRequest() and  buildLogoutRequest() ?

Reply:
we use the same method test both AuthnRequest and LogoutRequest